task/init/0.2/README.md

```diff
@@ -0,0 +1,19 @@
+# init task
+
+Initialize Pipeline Task, include flags for rebuild and auth. Generates image repository secret used by the PipelineRun.
+
+## Parameters
+|name|description|default value|required|
+|---|---|---|---|
+|image-url|Image URL for build by PipelineRun||true|
+|rebuild|Rebuild the image if exists|false|false|
+|skip-checks|Skip checks against built image|false|false|
+|pipelinerun-name|unused, should be removed in next task version|""|false|
+|pipelinerun-uid|unused, should be removed in next task version|""|false|
+
+## Results
+|name|description|
+|---|---|
+|build|Defines if the image in param image-url should be built|
+|container-registry-secret|unused, should be removed in next task version|
```

Comment:
> container-registry-secret|unused, should be removed in next task version|

Maybe we could drop this one too while we're at it?

Reply:
Seems like we could, good catch

Reply:
there is also

```
|pipelinerun-name|unused, should be removed in next task version|""|false|
|pipelinerun-uid|unused, should be removed in next task version|""|false|
```
I guess it's safe to remove those as well, right @mmorhun ?

Reply:
Yes, it should be safe, but will require more migration. Let's remove unused stuff.